app/models/page_content_register.rb

```diff
@@ -17,7 +19,10 @@ def new_blocks_attributes
                         else
                           @new_blocks_attributes
                         end
-    return @new_blocks_attributes if home_page? || footer?
+
+    if home_page? || footer?
+      return blocks_attributes + srcset_blocks(blk_attrs: blocks_attributes)
```

Comment:
I have had a look at it and played around with changing the code to eliminate the summation. Nonetheless, I can't figure out what's wrong with adding these two arrays; _block_attributes_ and _srcset_blocks_. If there is a better alternative I'm all ears to receive further feedback on this.